Tests/Repository/Vcs/GitDriverTest.php

```diff
@@ -102,6 +102,66 @@ public function testPublicRepositoryWithEmptyComposer($type, $filename)
      * @param string $type
      * @param string $filename
      */
+    public function testPublicRepositoryWithSkipUpdate($type, $filename)
+    {
+        $repoUrl = 'https://github.com/fxpio/composer-asset-plugin.git';
+        $identifier = '92bebbfdcde75ef2368317830e54b605bc938123';
+        $io = $this->getMockBuilder('Composer\IO\IOInterface')->getMock();
+
+        $this->config->merge([
+            'config' => [
+                'fxp-asset' => [
+                    'git-driver' => [
+                        'skip-update' => '1 week',
+                    ],
+                ],
+                'cache-vcs-dir' => sys_get_temp_dir().'/composer-test-cache-vcs',
+            ],
+        ]);
+
+        $repoConfig = array(
+            'url' => $repoUrl,
+            'asset-type' => $type,
+            'filename' => $filename,
```

Comment:
Please have a look at: https://github.com/fxpio/composer-asset-plugin/pull/278/commits/e676a76316fef9e9a2b3591f797d0975d26b1a79

I am getting...

```
8) Fxp\Composer\AssetPlugin\Tests\Repository\Vcs\GitDriverTest::testPublicRepositoryWithFilesystemCache with data set #1 ('bower', 'bower.json')
Trying to configure method "getConfig" which cannot be configured because it does not exist, has not been specified, is final, or is static
```

I thought we'd avoid that by extending from `AssetRepositoryManager` - I am getting confused :)

Also the `GitHubDriver` test now complains about 

```
18) Fxp\Composer\AssetPlugin\Tests\Repository\Vcs\GitHubDriverTest::testNoApi with data set #5 ('bower', 'bower.json', array('0123456789abcdef0123456789abc...234567'), array('master 0123456789abcdef012345...omment'))
Error: Call to a member function get() on null
```

I am unsure if I'd need to mock the config or add actual checks to the `GitDriver.php`